tests/test_enable_api.py

```diff
@@ -0,0 +1,100 @@
+# runs normally
+# but the logging system records everything in the first log(gems_enable.log) when use_gems (with different paths)
+# it seems to be related to FileHandler.
+import flag_gems
+import pytest
+import torch
+import time
+import os
+import re
+
+@pytest.mark.enable
+def test_enable():
+   # flag_gems.enable(
+   #    record=True,
+   #    path="./gems_enable.log"
+   # )
+
+   with flag_gems.use_gems(record=True, path="./gems_enable.log"):
+      a = torch.tensor([1.0, 2.0, 3.0], device=flag_gems.device)
+      b = torch.tensor([4.0, 5.0, 6.0], device=flag_gems.device)
+      c = a + b
+      d = a * b
+      e = flag_gems.sum(a)
+      f = torch.sum(a)
+
+      print(f"add result: [{c}]")
+      print(f"mul result: [{d}]")
+      print(f"sum result flaggems: [{e}]")
+      print(f"sum result torch: [{f}]")
```

Comment:
This test case is not asserting anything, why?
Test cases are supposed to be executed silently without "polluting" the console ... shall we remove these `print` calls?